composer.json

```diff
@@ -31,6 +31,11 @@
             "Sabre\\Uri\\" : "lib/"
         }
     },
+    "autoload-dev": {
```

Comment:
It seems this wasnt required before.... why?

Reply:
It uses the ```autoload-dev``` in ```composer.json``` to load ```Tests``` classes automatically.